internal/debug/flags.go

```diff
@@ -77,6 +76,17 @@ var (
 		Usage:    "Write logs to a file",
 		Category: flags.LoggingCategory,
 	}
+	backtraceAtFlag = &cli.StringFlag{
+		Name:     "log.backtrace",
+		Usage:    "Request a stack trace at a specific logging statement (e.g. \"block.go:271\")",
+		Value:    "",
+		Category: flags.LoggingCategory,
+	}
```

Comment:
Move htem into `flags_legacy.go`, where other deprecated flags reside. Belonging to `flags.DeprecatedCategory`.

Also add a note `(deprecated)` to the Usage-string

Reply:
wondering where would be a proper place to log a warning?  if I put them in `flags_legacy.go`, I cannot import into `internal/debug/flags.go` without creating an import cycle.